core/templates/expressions/expression-syntax-tree.service.ts

```diff
@@ -35,6 +35,7 @@ interface EnvDict {
 }
 
 export class ExpressionError extends Error {
+  // 'message' is optional beacuse we may not want a custom error message.
```

Comment:
Not the full story, `message` is optional because that is the same interface provided by real `Error` classes.

Reply:
done